src/jdmtool/skybound.py

```diff
@@ -1,8 +1,38 @@
 from typing import Iterable
 
+import logging
 import usb1
 
 
+log = logging.getLogger(__name__)
+
+KNOWN_FIRMWARE_RESPONSES={
+    "20071203": {
+        "responses": [
+            b"\x12\x01\x10\x01\xFF\x83\xFF\x40\x39\x0E\x50\x12\x00\x00\x00\x00\x00\x01",
+            b"\x09\x02\x20\x00\x01\x01\x00\x80\x0F",
+            b"\x09\x02\x20\x00\x01\x01\x00\x80\x0F\x09\x04\x00\x00\x02\x00\x00"
+            b"\x00\x00\x07\x05\x81\x02\x40\x00\x05\x07\x05\x02\x02\x40\x00\x05"
+        ]
+    },
+    "20140530": {
+        "responses": [
+            b"\x12\x01\x10\x01\xFF\x83\xFF\x40\x39\x0E\x50\x12\x00\x00\x00\x00\x00\x01",
+            b"\x09\x02\x20\x00\x01\x01\x00\x80\x32",
+            b"\x09\x02\x20\x00\x01\x01\x00\x80\x32\x09\x04\x00\x00\x02\x00\x00"
+            b"\x00\x00\x07\x05\x81\x02\x40\x00\x05\x07\x05\x02\x02\x40\x00\x05"
+        ]
+    }
+}
```

Comment:
I originally captured USB traffic using Wireshark and used `usbrply` to generate Python code, and left all the requests/responses hard-coded just in case they were required to initialize the device. But, it turns out, these are completely standard "get configuration" requests. E.g., the difference in these responses - `\x0F` vs `\x32` - means the device supports 30mA vs 100mA.

So these are mostly irrelevant to jdmtool; I will try to use some higher-level libusb function to do a sanity check, but otherwise ignore most of these bytes.

Reply:
Oh wow. Turn out, these requests were not made by JDM, or even by Windows... They're from Linux itself. So they most certainly don't belong in jdmtool. Oops.